docs/architecture/introduction.rst

```diff
@@ -367,25 +366,26 @@ If you are using static resource pools and launching agents by hand, you will ne
 Migrate to Resource Pools
 -------------------------
 
-With the introduction of resource pools, the :ref:`master-config-reference` format has changed to a
-new format.
-
-This is a backwards compatible change and cluster configurations in the old format will continue to
-work. A configuration in the old format is interpreted as a cluster with a single resource pool that
-is the default for both CPU and GPU tasks. However, to take full advantage of resource pools, you
-will need to convert to the new format, which is a simple process of moving around and renaming a
-small number of top-level fields.
-
-The old format had the top level fields of ``scheduler`` and ``provisioner`` which set the scheduler
-and provisioner settings for the cluster. The new format has the top level fields
-``resource_manager`` and ``resource_pools``. The ``resource_manager`` section is for cluster level
-setting such as which pools should be used by default and the default scheduler settings. The
-``scheduler`` information is identical to the ``scheduler`` field in the legacy format. The
-``resource_pools`` section is a list of resource pools each of which has a name, description and
-resource pool level settings. Each resource pool can be configured with a ``provider`` field that
-contains the same information as the ``provisioner`` field in the legacy format. Each resource pool
-can also have a ``scheduler`` field that sets resource pool specific scheduler settings. If the
-``scheduler`` field is not set for a specific resource pool, the default settings are used.
+Resource pools were introduced with Determined 0.14.0 resulting in changes to the
+:ref:`master-config-reference` format.
```

Comment:
```suggestion
:ref:`master configuration <master-config-reference>` format.
```

Reply:
Could you also update the link to `agent-config-reference` above similarly?

Reply:
done